### ISSUE TITLE
object-curly-spacing: comments should not trigger rule

### ISSUE THREAD

**Issue by silverwind** [eslint/eslint#12089]
**Tell us about your environment**

* **ESLint Version:** 6.1.0
* **Node Version:** 12.7.0
* **npm Version:** 6.10.0

**Please show your full configuration:**

```yaml
rules:
  - object-curly-spacing: [2, never]
```

**What did you expect to happen?**

This code to pass:

```js
const a = { // comment
  a: 1,
};
```

**What actually happened? Please include the actual, raw output from ESLint.**

```
1:11  error  There should be no space after '{' object-curly-spacing
```

Maybe related to https://github.com/eslint/eslint/commit/67c08b67509c54acd96aab2cec22efb53bfe6265#diff-9da030a8c7c9a4eea24f42982a0391b0.

**Are you willing to submit a pull request to fix this bug?**

no

**Comment by mdjermanovic** [comment]
This is same as #11902

**Comment by platinumazure** [comment]
Closing this as a duplicate of #11902, please track that issue for resolution. Thanks!

**Comment by silverwind** [comment]
Indeed, it is.
